src/rohiniBDec19/Assignment2/Program1.java

```diff
@@ -0,0 +1,28 @@
+package rohiniBDec19.Assignment2;
+
+/*1) Find the missing number in integer array of 1 to 10? */
+
+public class Program1 {
+	 int findMissing(int a[])
+	   {
+		
+		int actualSum=0;
+		   for(int i=0;i<a.length;i++)
+		   {
+			   actualSum=actualSum+a[i];  
+				
+			}
+		   return actualSum;
+		}
+		
+		 public static void main(String[] args) {
+			Program1 pgrm = new Program1();
+			int sum=55; //sum 1 to 10
```

Comment:
you can not hard code this value you have to do this sum using for loop and need make it dynamic

Reply:
Thanks Harsh.  Got it will make dynamic for this kind of programs n will make sure not pass hardcode value